.github/workflows/ci_test-full.yml

```diff
@@ -56,6 +56,7 @@ jobs:
     env:
       PYTEST_ARTEFACT: test-results-${{ matrix.os }}-py${{ matrix.python-version }}-${{ matrix.requires }}.xml
       PYTORCH_URL: https://download.pytorch.org/whl/cpu/torch_stable.html
+      TRANSFORMERS_CACHE: .cache/huggingface/
```

Comment:
```suggestion
```
this is not needed

Reply:
It was apparently needed to get some of the Bert score tests unstuck

Reply:
strange as this has to fail on Win :)